src/plugins/data/common/search/aggs/buckets/histogram_fn.ts

```diff
@@ -72,7 +72,7 @@ export const aggHistogram = (): FunctionDefinition => ({
       }),
     },
     interval: {
-      types: ['string'],
+      types: ['number', 'string'],
```

Comment:
We had incorrectly typed this as `string` only (since you can do `auto`), but in most cases the interval is actually a number when explicitly provided. Keeping this as a string caused some functional test failures where an incorrect interval got applied when the histogram expression function cast the provided number to a string instead of keeping its type.